Source/KakapoDB.swift

```diff
@@ -92,24 +92,32 @@ public class KakapoDB {
         return object
     }
     
-    public func update<T: Storable>(entity: T) -> Bool {
-        return barrierSync {
+    public func update<T: Storable>(entity: T) throws {
+        let updated: Bool = barrierSync {
```

Comment:
I think we can make it cleaner by having a retrow in the barrierSync and just throw in the guard.